BuildingBlocks/test/BuildingBlocks.Application.Tests/Mediatr/QuotaEnforcerBehaviorTests.cs

```diff
@@ -75,55 +31,11 @@ public async void Succeeds_when_none_of_the_metrics_exist_in_the_repository()
         nextMock.WasCalled.Should().BeTrue();
     }
 
-    [Fact]
-    public async void Succeeds_when_none_of_the_metrics_are_exhausted()
-    {
-        // Arrange
-        var behavior = CreateQuotaEnforcerBehavior(
-            TestData.MetricStatus.ThatWasExhaustedUntil2DaysAgo,
-            TestData.MetricStatus.ThatWasExhaustedUntilYesterday
-        );
-        var nextMock = new NextMock<TestData.IResponse>();
-
-        // Act
-        Func<Task> acting = async () => await behavior.Handle(
-            new TestData.TestCommand(),
-            nextMock.Value,
-            CancellationToken.None);
-
-        await Task.Run(acting, CancellationToken.None);
-
-        // Assert
-        nextMock.WasCalled.Should().BeTrue();
-    }
-
-    [Fact]
-    public void Throws_QuotaExhaustedException_when_the_metric_is_still_exhausted()
-    {
-        // Arrange
-        var behavior = CreateQuotaEnforcerBehavior(TestData.MetricStatus.ThatIsExhaustedFor10Days);
-        var nextMock = new NextMock<TestData.IResponse>();
-
-        // Act
-        Func<Task> acting = async () => await behavior.Handle(
-            new TestData.TestCommand(),
-            nextMock.Value,
-            CancellationToken.None);
-
-        // Assert
-        var exceptionExhaustedMetrics = acting.Should().AwaitThrowAsync<QuotaExhaustedException>().Which.ExhaustedMetricStatuses;
-        exceptionExhaustedMetrics.Should().HaveCount(1);
-        exceptionExhaustedMetrics.First().MetricKey.Should().Be(TestData.MetricStatus.ThatIsExhaustedFor10Days.MetricKey);
-    }
-
     [Fact]
     public void Throws_QuotaExhaustedException_when_exactly_one_metric_is_exhausted()
     {
         // Arrange
-        var behavior = CreateQuotaEnforcerBehavior(
-            TestData.MetricStatus.ThatIsExhaustedFor1Day,
-            TestData.MetricStatus.ThatWasExhaustedUntilYesterday
-        );
+        var behavior = CreateQuotaEnforcerBehavior(TestData.MetricStatus.ThatIsExhaustedFor1Day);
         var nextMock = new NextMock<TestData.IResponse>();
```

Comment:
You can inline this, because you don't do any assertions on it anyway. This reduces noise.